src/meander/util/epsilon.cljc

```diff
@@ -430,3 +430,30 @@
                    sym))
                (symbol (name (:name cljs-ns)) (name sym)))
              sym)))
+
```

Comment:
I'm not proud of any of this but I don't know of a cleaner way and I don't feel like spending more time on it. I'm open to improvements here if anyone has some.